dask_expr/collection.py

```diff
@@ -871,10 +871,12 @@ def optimize(collection, fuse=True):
     return new_collection(expr.optimize(collection.expr, fuse=fuse))
 
 
-def from_pandas(data, *args, **kwargs):
+def from_pandas(data, npartitions=1, sort=True, **kwargs):
```

Comment:
Do we need `kwargs` now that we are already exposing all "public" options explicitly.

Reply:
Probably not, removed